test/test_automl/test_estimators.py

```diff
@@ -704,3 +710,37 @@ def test_check_askl2_same_arguments_as_askl():
                            'metadata_directory']
     unexpected_args = set(extra_arguments) - set(expected_extra_args)
     assert len(unexpected_args) == 0, unexpected_args
+
+
+# which should return self actually return self.
```

Comment:
Sorry, could you please clarify this comment? I fail to understand it :(

Reply:
sorry, I think this was a problem with rebase, which I have now fixed.